lib/openstates/api.rb

```diff
@@ -15,6 +15,11 @@ def bills(options = {})
         end
       end
 
+      if options[:id]
+        url << "#{options[:id]}/"
+        options.delete(:id)
+      end
```

Comment:
I'm not sure this is actually necessary. If you look at the implementation of the `bills` method in `openstates/api.rb`, the bill id should be passed in as part of the options hash.

![screen shot 2015-07-01 at 1 17 25 pm](https://cloud.githubusercontent.com/assets/997161/8460602/db56b532-1ff3-11e5-8cb7-128acd7bd307.png)

The documentation for the openstates api shows needing the state, the session, and the bill id to get bill details *OR* just the bill id. I think the change we really want is to check for the presence of just the bill id or all three and build the correct URL that way. Thoughts?